Code/Common/include/Ancillary/type_list2.h

```diff
@@ -139,25 +139,10 @@ struct index_of<typelist<t0, Ts...>, T>
  */
 template <typename Typelist, typename T>
 struct has_type;
-#if !defined(_MSC_VER) || _MSC_VER >= 1930
-// note the following causes a syntax error with MS VS 16
 template <typename... Ts, typename T>
 struct has_type<typelist<Ts...>, T>
-  : std::integral_constant<bool, std::max<bool>(std::initializer_list<bool>{ std::is_same<Ts, T>::value... })>
+  : std::integral_constant<bool, ((std::is_same<Ts, T>::value ) || ...) >
```

Comment:
@blowekamp I find the original `std::max(std::initializer_list{...})` very clever 😺, but the fold expression is even better 👍